sixpack.js

```diff
@@ -199,7 +199,15 @@
         } else {
             var httpModule = url.startsWith('https') ? 'https' : 'http';
             var http = eval('require')(httpModule); // using eval to skip webpack bundling and warnings
-            var req = http.get(url, { headers: { 'Cookie': cookie } }, function(res) {
+
+            var parsedUrl = new URL(url);
+            var options = {
+                port: parsedUrl.port,
```

Comment:
Add protocol
```suggestion
                port: parsedUrl.port,
                protocol: parsedUrl.protocol,
```

Reply:
Since we have this:
```javascript
var httpModule = url.startsWith('https') ? 'https' : 'http';
var http = eval('require')(httpModule); // using eval to skip webpack bundling and warnings
//...
http.get()
```
Do we really need `protocol`?

Reply:
It depends of the module implementation if it will use the correct protocol if the value is empty, to be explicit and avoid needing to rely on default value, I prefer to add the value here

Reply:
It works without specify the protocol, but we could be more confident including it

Reply:
Done.